index.html

```diff
@@ -45,6 +203,42 @@ <h2>I'm Andrea.<br>Glad to see you!</h2>
               </li>
             </ul>
           </div>
+          <button class="efecbot">Get my resume</button>
```

Comment:
- [ ] Great job adding the hover, & active styles :rocket:, but in order to match the Figma design the text color should change to white when hovered or pressed. Kindly, include these styles in your button.
![Screenshot from 2023-04-29 04-49-30](https://user-images.githubusercontent.com/86069740/235278062-da07bf04-4b8b-4b9d-b41b-8bc9728e715c.png)